index.js

```diff
@@ -101,13 +101,13 @@ var patchProperty = (node, key, oldValue, newValue, listener, isSvg) => {
 }
 
 var createNode = (vdom, listener, isSvg) => {
-  var props = vdom.props
+  var {is, ...props} = vdom.props
   var node =
     vdom.type === TEXT_NODE
       ? document.createTextNode(vdom.tag)
       : (isSvg = isSvg || vdom.tag === "svg")
-      ? document.createElementNS(SVG_NS, vdom.tag, { is: props.is })
-      : document.createElement(vdom.tag, { is: props.is })
+      ? document.createElementNS(SVG_NS, vdom.tag, is && {is})
+      : document.createElement(vdom.tag, is && {is})
```

Comment:
Sorry for the late response. 

Perhaps `props.is && props` would be fewer bytes and work just as well. What do you think?

Reply:
I’ll give it a try!

Reply:
Good call – saves 8 bytes (minified and gzipped)!  :)

Reply:
And works just as well!